en/docs/develop/enable-verification-for-updated-mobile-number.md

```diff
@@ -64,6 +64,25 @@ When a user updates their mobile number in the user profile, an SMS OTP is sent
         enable_verification = true
         verification_sms_otp_validity = “5”
         ```
+    3. By default mobile number verification is not allowed for privileged users. Add the following property to the above `deployment.toml` config to enable this server wide.
+        ```toml
+        [identity_mgt.user_claim_update.mobile]
+        enable_verification_by_privileged_user = true
+        ```
+
+    4. Add the following properties to the `deployment.toml` file to restrict resending verification codes only by privileged users.
+        ```toml
+        [[resource.access_control]]
+        context = "(.*)/api/identity/user/v1.0/resend-code(.*)"
+        secure = "true"
+        http_method = "all"
+        permissions=["/permission/admin/manage/identity/identitymgt"]
+        scopes=["internal_identity_mgt_view","internal_identity_mgt_update","internal_identity_mgt_create","internal_identity_mgt_delete"]
+        ```
+
+        The mobile number verification by a privileged user is available as an update in WSO2 IS 5.11.0 from update level 150 onwards (Updates 2.0 model). If you don't already have this update, see the instructions on [updating WSO2 products](https://updates.docs.wso2.com/en/latest/updates/overview/).
```

Comment:
!!! info
    Mobile number verification by a privileged user is available as an update in WSO2 IS 5.11.0 from update level 150 onwards (Updates 2.0 model). If you don't already have this update, see the instructions on [updating WSO2 products](https://updates.docs.wso2.com/en/latest/updates/overview/).

Reply:
Let't put this to the beginning of the note

Reply:
Addressed with d9428901276377ae3f901fdab573f5006f7846ff